Yoast/javascript
# Add transition words assessment for Hebrew

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Implements the transition words assessment for Hebrew.
* [Yoast SEO Free] Adds the assessment for transition words in Hebrew.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Run yarn test and make sure all tests still pass.
* Add more specs and check that all of them work.

In the plugin:

* Set your environment to Hebrew
* Paste in this text containing the transition word `גם`:
` .העיר מקודשת ביהדות, בנצרות ובאסלאם. גם היא הייתה מרכז חיי העם היהודי בימי קדם, ומושא כמיהתו וכיסופיו בעת גלותו. `
* The assessment bullet should be green and read: `Link words: Well done!`
* Out of the two sentences, only the second should be highlighted.


## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/browse/LIN-513
